File: javascript/1188.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var valores = input.split('\n'), count = 0, sum = 0;
var oper = String(valores.shift()).trim();

for (let i = 0; i < 12; i++) {
    for (let j = 0; j < 12; j++) {
        var val = parseFloat(valores.shift());
        if ((i > 6) && (11 - i < j && j < i)) {
            sum += val;
            count++;
        }
    }
}
console.log(oper == 'S' ? sum.toFixed(1) : (sum / numeros).toFixed(1))

File: C/1188.c
#include <stdio.h>

int main() {
    int count = 0, i, j;
    float val, sum = 0;
    char oper;
    scanf("%c", &oper);
    
    for (i = 0; i < 12; i++) {
        for (j = 0; j < 12; j++) {
            scanf("%f", &val);
            if ((i > 6) && (11 - i < j && j < i)) {
                sum += val;
                count++;
            }
        }
    }
    printf("%.1f\n", oper == 'S' ? sum : sum / count);

    return 0;
}